### PR TITLE
test(universe): validate schema mutations null support

### LINKED ISSUE #656: Update schema functions to understand Null

### PR DESCRIPTION
**Pull request by affo**
Fixes #656 

### Done checklist
- [x] docs/SPEC.md updated
- [x] Test cases written

### CODE REVIEW

File: stdlib/universe/schema_functions_test.go [hunk 1]
```diff
@@ -876,6 +876,60 @@ func TestDropRenameKeep_Process(t *testing.T) {
 				),
 			}},
 		},
+		{
+			name: "duplicate group key",
```
Comment (affo) [inline stdlib/universe/schema_functions_test.go]: The behavior of duplicating a group key was not defined. See the addition to SPEC.

File: stdlib/universe/schema_functions_test.go [hunk 2]
```diff
@@ -876,6 +876,60 @@ func TestDropRenameKeep_Process(t *testing.T) {
...
+					{Label: "2a", Type: flux.TFloat},
+					{Label: "3a", Type: flux.TFloat},
+				},
+				Data: [][]interface{}{
+					{1.0, 2.0, 3.0},
+					{1.0, 12.0, 3.0},
+					{1.0, 22.0, 3.0},
+				},
+				KeyCols:   []string{"1a", "3a"},
+				KeyValues: []interface{}{1.0, 3.0},
```
Comment (wolffcm) [inline stdlib/universe/schema_functions_test.go]: (nit) the `keyValues` are unnecessary if you have data in the table, so omitting this line would make the test a tiny bit more concise.

File: stdlib/universe/schema_functions_test.go [hunk 3]
```diff
@@ -876,6 +876,60 @@ func TestDropRenameKeep_Process(t *testing.T) {
...
+				GroupKey: execute.NewGroupKey(
```
Comment (wolffcm) [inline stdlib/universe/schema_functions_test.go]: Likewise, the `executetest.ProcessTestHelper` infrastructure should create the `GroupKey` for you, so you don't need to define it here.  For table that has data in it, on `KeyCols` needs to be defined.

Comment (affo) [reply stdlib/universe/schema_functions_test.go]: Great, thank you. It was copy paste from other tests in the same file. Anyhow, thanks for the clarification!

**Pull request merged**